front/migrations/20240423_backfill_customerio.ts

```diff
@@ -31,7 +30,14 @@ const backfillCustomerIo = async (execute: boolean) => {
             }),
             // NOTE: this is unrelated to customerio, but leveraging this backfill
             // to also identify all users on Amplitude.
-            AmplitudeServerSideTracking._identifyUser({ user: u }),
+            AmplitudeServerSideTracking._identifyUser({
+              user: {
+                ...u.toJSON(),
+                fullName: `${u.firstName} ${u.lastName}`,
```

Comment:
Why is this needed?

Reply:
Because ts is unhappy with it, as u is a model and not a resource
(Also that's just a backfill migration so it was a quickfix not trying to make it too clean)